README.md

```diff
@@ -51,15 +51,20 @@ The page builder should now be running at `http://localhost:3000`.
 To deploy the page builder to a production environment, run the following command:
 
 ```bash
-npm run build
+pnpm run build
```

Comment:
pnpm run ???

Reply:
Ow, my bad. I forgot to remove it here too

Reply:
Fixed